### PR TITLE
PRO-5526: Decode name params

### PR DESCRIPTION
**Pull request by meganaslattery**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRO-5526

### Change description ###

Allow executors to be notified with accents in names

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/probate/services/invitation/controllers/InvitationController.java [hunk 1]
```diff
@@ -45,6 +52,18 @@ public String invite(@Valid @RequestBody Invitation invitation,
         return linkId;
     }
 
+    private Invitation decodeURL(Invitation invitation) throws UnsupportedEncodingException {
+        invitation.setExecutorName(decodeURLParam(invitation.getExecutorName()));
+        invitation.setFirstName(decodeURLParam(invitation.getFirstName()));
+        invitation.setLastName(decodeURLParam(invitation.getLastName()));
```
Comment (davejones74) [inline src/main/java/uk/gov/hmcts/probate/services/invitation/controllers/InvitationController.java]: Hi Megan, can you move this to the Invitation service and rename decodeURL to something like decodeInvitation, can you also move decodeURLParam, thanks.

File: src/main/java/uk/gov/hmcts/probate/services/invitation/model/Invitation.java [hunk 2]
```diff
@@ -34,10 +34,14 @@ public String getFirstName() {
         return firstName;
     }
 
+    public void setFirstName(String firstName) { this.firstName = firstName; }
```
Comment (davejones74) [inline src/main/java/uk/gov/hmcts/probate/services/invitation/model/Invitation.java]: Can you please just update the style to use carridge returns so it's consistence. I.e. 
public String getLastName() {
    return lastName;
}

Comment (meganaslattery) [reply src/main/java/uk/gov/hmcts/probate/services/invitation/model/Invitation.java]: Done :)

File: src/main/java/uk/gov/hmcts/probate/services/invitation/controllers/InvitationController.java (hunk 1 above)
Comment (meganaslattery) [reply src/main/java/uk/gov/hmcts/probate/services/invitation/controllers/InvitationController.java]: Done :)

**Pull request closed without merging**